pkg/plugin/plugin.go

```diff
@@ -240,3 +240,27 @@ func (p *plugin) GetControlPlanePods(ctx context.Context, oc *api.OpenShiftManag
 	}
 	return json.Marshal(pods)
 }
+
+func (p *plugin) ForceUpdate(ctx context.Context, cs *api.OpenShiftManagedCluster, deployFn api.DeployFn, pluginTemplate *pluginapi.Config) *api.PluginError {
```

Comment:
do you need to pass the `pluginapi.Config` it should be already available via `p.Config`

Reply:
i've taken out `pluginapi.Config` however it is not available via `p.Config` ;)